docs/disciplines/Data_Science/Data_Curation/Clinical_Trial_Curation/index.md

```diff
@@ -0,0 +1,130 @@
+# Clinical Trial Curation
+
+## Immunotherapy datasets
+### Introduction
+This documentation goes over the clinical trial data curation process in detail, using immunotherapy data.
+
+### Objective
+The objective is to curate a clinical dataset into R's [MultiAssayExperiment](https://bioconductor.org/packages/release/bioc/html/MultiAssayExperiment.html) object. An example of a clinical data MultiAssayExperiment (MAE) object can be found in [ORCESTRA](https://www.orcestra.ca/clinical_icb/10.5281/zenodo.7004828). 
+
+Currently, a clinical data object contains the following data parts:
+
+1. Clinical metadata: Contains patient/sample metadata.
+2. Molecular profiles: Molecular assay data (Currently RNA-seq, SNV or CNA) which is formatted in either [RangedSummarizedExperiment or regular SummarizedExperiment object](https://bioconductor.org/packages/devel/bioc/vignettes/SummarizedExperiment/inst/doc/SummarizedExperiment.html).
+
+### Data Access
+#### Public data
+
+If the source is Pubmed, the raw omics files and clinical response metadata are available from Supplementary or external repository links in Data Availability section of the paper.
+
+#### Private data
+
+Private data such as PHI, clinical response might be available only upon request. Please contact the author(s) or whoever is responsible for requesting such data.
+
+### Data Processing Overview
+![](img/Clinical_trial_curation_overview.png){: align=left height=25% width=25% }
+
+An example of clinical data processing pipeline can be found here as [a Snakemake pipeline](https://github.com/BHKLAB-DataProcessing/ICB_Braun-snakemake/blob/main/Snakefile). 
+
+Generally, an overall process of the curation follows the steps outlined below:
+
+1. **Download source data**: Download data from publications or data repositories. The source data can be in various formats such as an Excel file, CSV or TXT. 
+2. **Process raw molecular data, if available**: The RNA-seq processing from raw FASTQ is outlined on the [RNAseq raw processing page].
```

Comment:
Was the RNASeq page supposed to point to an existing handbook page?

Reply:
It will in the future. The page hasn't been migrated yet, but it exists [on Confluence](https://collaborate.uhnresearch.ca/confluence/display/BHKLabPRC/RNA+seq+raw+processing).

I'll make an issue to migrate this page and update the link here once it's done.

Reply:
Issue created: https://github.com/bhklab/handbook/issues/110